easybuild/easyconfigs/o/OpenBLAS/OpenBLAS-0.3.27-GCC-13.3.0.eb

```diff
@@ -0,0 +1,46 @@
+name = 'OpenBLAS'
+version = '0.3.27'
+
+homepage = 'http://www.openblas.net/'
```

Comment:
can use https

Reply:
Good catch, I am going to update the MR.